be/api/po-domain/src/main/java/cz/covid/po/api/domain/repository/HealthCheckResultRepository.java

```diff
@@ -0,0 +1,13 @@
+package cz.covid.po.api.domain.repository;
+
+import cz.covid.po.api.domain.model.Address;
+import cz.covid.po.api.domain.model.HealtCheckResult;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface HealthCheckResultRepository extends JpaRepository<HealtCheckResult, Long> {
+
+    List<HealtCheckResult> findAllByResultSentAtIsNull();
```

Comment:
Logika predpoklada, ze tento datun je null, kdyz k poslani SMS nedoslo a pak se to nastavuje na `OffsetDateTime.now()`.